### PR TITLE
PIMS-1877 Updating express-api

### PR DESCRIPTION
**Pull request by TaylorFries**
## 🎯 Summary

  
[PIMS-1877: ](https://citz-imb.atlassian.net/browse/PIMS-1877)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.

### CODE REVIEW

File: express-api/package.json [hunk 1]
```diff
@@ -38,10 +38,12 @@
     "pg": "8.12.0",
     "reflect-metadata": "0.2.1",
     "swagger-ui-express": "5.0.0",
+    "ts-node": "10.9.1",
+    "tsconfig-paths": "4.2.0",
     "typeorm": "0.3.20",
     "typeorm-naming-strategies": "4.1.0",
     "winston": "3.13.0",
-    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz",
+    "xlsx": "0.18.5",
```
Comment (dbarkowsky) [inline express-api/package.json:46]: [Same comment as last time](https://github.com/bcgov/PIMS/pull/2521#discussion_r1676221873). Is there a way to keep this from coming up in the dependency reports?

**Review by dbarkowsky — CHANGES_REQUESTED** [review]

File: .github/helpers/parse-json5-config.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ fs.readFile(filePath, "utf8", (err, data) => {
 
     // Set each key-value pair as an environment variable
     for (const [key, value] of Object.entries(jsonData)) {
+      console.log("test key: ", key, " Value: ", value)
```
Comment (dbarkowsky) [inline .github/helpers/parse-json5-config.js:63]:
```suggestion
```
Do we want this?

**Review by dbarkowsky — APPROVED** [review]
Approved. Remove the console log if you didn't mean to have that in your script.

File: .github/helpers/parse-json5-config.js (hunk 2 above)
Comment (TaylorFries) [reply .github/helpers/parse-json5-config.js]: removed

File: express-api/package.json (hunk 1 above)
Comment (TaylorFries) [reply express-api/package.json]: couldnt get the ignorePackages to work, instead included a blurb on the Jira ticket about xlsx

**Pull request merged**